kaldb/pom.xml

```diff
@@ -487,7 +492,14 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>2.4.1</version>
+                <version>3.2.4</version>
+                <dependencies>
+                    <dependency>
```

Comment:
Do we need these external dependencies? Can we make this change without these additional dependencies?

Reply:
How hard would it be to emit log messages in span json format?

Reply:
nope :) It won't compile. I've linked the github issue